src/PointwiseFunctions/EquationsOfState/Tags.hpp

```diff
@@ -0,0 +1,24 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "Options/Options.hpp"
+#include "PointwiseFunctions/Hydro/EquationsOfState/EquationOfState.hpp"
+
+namespace OptionTags {
+/// \ingroup OptionTagsGroup
+/// Can be used to retrieve the equation of state from the cache without having
+/// to know the template parameters of EquationOfState.
+struct EquationOfStateBase {};
+
+/// \ingroup OptionTagsGroup
+/// The equation of state, with `IsRelativistic` and `ThermodynamicDim` left
+/// as template parameters.
+template <bool IsRelativistic, size_t ThermodynamicDim>
+struct EquationOfState : EquationOfStateBase {
```

Comment:
I think these tags exist now if you do a rebase, just in `Hydro/Tags.hpp`

Reply:
I believe the tag I provide makes it possible to provide the equation of state at runtime, while the tag in Hydro/Tags will require the EoS to be provided in the metavars - is my understanding correct? How should I decide whether I should make an option specifiable at run time vs compile time?

Reply:
I think we can now use the general tags, no?